### PR TITLE
Added contribute.mdx page and images, added it to the left menu

### PR DESCRIPTION
**Pull request by Simonbdy**
contribute.mdx: new content on contribution
img/: 7 new images
sidebarsDocs.js: added contribute.mdx to the left menu in documentation

⚠ PLEASE DO NOT DELETE THE TEXT BELOW ⚠

## Pull request's description

*Add here a description of the modifications you made.*

## Checklist before merging (please do not edit)
You must review you work before to submit the review to others.

### Self-review of your content
Remember the content must be readable and understandable by someone else than yourself.
- [x] From a technical point of view.
- [x] From a grammatical point of view (spelling mistakes, typos, clarity, etc.), and following the guidelines at the bottom.

### External reviews of your content
- [x] You requested a technical review.
- [x] You requested a grammatical review.
- [ ] You validated with @K0rdan or @alexgeron-Luos that it is safe to merge.

### Some guidelines to keep in mind
- Colons (:), semi-colons (;), exclamation (!), and interrogation points (?) are not preceded by a space (like full stops and commas). E.g.: Colons!
- File names and/or address are put in *italic*. E.g.: The file _main.c_.
- Functions, variables, or more generally short codes are put between grave accents. E.g.: To obtain `code()`, type \`code()\`.
- Long codes are put into blocks of code with three grave accent on each side, and the language's name:<br />
\`\`\`c<br />
// Some C language <br />
\`\`\`<br />

- "Luos engine" has a upper case on the **L** for "Luos" and lower case on the **e** for "engine".
- We call it "Luos engine" as a proper name, and ***not*** "the Luos engine".
- Following that fashion, anything that's owned by "Luos engine" implies that we must use `'s` as the standard English rule to show the possessive case, e.g. "Luos engine's API".
- The names pipe, gate, inspector, sniffer, app or application, driver, etc. have a lower case and can be refereed with the determiner `the`. E.g.: The inspector.

### CODE REVIEW

**Comment by Emanuel-Allely** [comment]
## **Is this a contributing guide or a PR template?**

In the Github community standards, there are 2 distinct and mandatory files called CONTRIBUTING.md and pull_request_template.md. I feel this file is a melt between these two files. Is this file a CONTRIBUTING.md or a pull_request_template.md?

## **If it is a CONTRIBUTING.md:**

CONTRIBUTING.md should display guidelines following these recommendations: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors (examples at the bottom).

https://github.com/atom/atom/blob/master/CONTRIBUTING.md
https://github.com/rails/rails/blob/main/CONTRIBUTING.md
https://github.com/opengovernment/opengovernment/blob/master/CONTRIBUTING.md

Last but not least, I think the CONTRIBUTING.md should make people **want** to contribute and be a nice and friendly document.

## **If it is a pull_request_template.md:**

It's clear. You may add a link to the dev channel on Discord, to invite the contributor to open a thread about his PR. This incite contributor to discuss with us about his contribution and create activity and engagement on the community.

## **Files examples**
You may find here good examples of the 6 files needed to match the community standards: https://github.com/Luos-io/luos_engine/community.

**Comment by Simonbdy** [comment]
@Emanuel-Allely it's neither.
It's a documentation page about contribution.

The contributing file's PR is being fixed here (many conflicts): https://github.com/Luos-io/luos_engine/pull/305
The PR and issues templates are yet to be improved, this week hopefully.

See this page as a documentation page for contribution, linked to the contribution guidelines CONTRIBUTING.mdx.

**Comment by Emanuel-Allely** [comment]
Ok, got it! So if it's a Docusaurus page, can you name it contribute-to-luos.mdx? Better for SEO :)

**Review by Emanuel-Allely — APPROVED** [review]
Pretty clear and perfect! GG bro

**Review by alexgeron-Luos — APPROVED** [review]

File: apps/documentation/docs/contribute-to-luos.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+As a contributor, you will sometimes open new issues, for example if you spot a typo, want to report a bug, or request a new feature. 
+
+But you will also be able to contribute by commenting in existing issues by bringing some inputs and discussing bug resolving or feature implementing with other contributors.
+
+#### Checklists
+
+**Before creating a new issue:**
+
+- [ ] Verify what you want to talk about: if you are reporting a bug, are you sure it is an actual bug and not a particular way of using Luos that doesn't suite you in a particular case? 
+- [ ] If you want to request a feature, did you check that there is no way to do what you want to do? Do not hesitate to search in the <a href="https://docs.luos.io/docs" target ="_blank">documentation</a> first and <a href="https://discord.gg/luos" target ="_blank">ask questions on the Discord</a> before to rush in the creation of a new issue.
```
Comment (K0rdan) [inline apps/documentation/docs/contribute-to-luos.mdx:42]: Change link from : 'https://docs.luos.io/docs' to 'https://www.luos.io/docs/luos-technology'

File: apps/documentation/sidebarsDocs.js [hunk 2]
```diff
@@ -231,5 +231,8 @@ module.exports = {
         },
       ],
     },
+      {type: 'doc',
```
Comment (K0rdan) [inline apps/documentation/sidebarsDocs.js:234]: You should setup your linter

**Review by K0rdan — CHANGES_REQUESTED** [review]
PNG files should be in a higher definition or custom

**Comment by Simonbdy** [comment]
@K0rdan thanks for the review, I fixed your suggestions

**Review by K0rdan — APPROVED** [review]

**Pull request merged**